src/desktop/apps/article/components/InfiniteScrollNewsArticle.tsx

```diff
@@ -129,7 +129,8 @@ export class InfiniteScrollNewsArticle extends Component<Props, State> {
   }
 
   renderWaypoint = () => {
-    const { isEnabled, isLoading, error } = this.state
+    const { isLoading, error } = this.state
+    const isEnabled = !sd.IS_GOOGLEBOT && this.state.isEnabled
```

Comment:
we'll want to use our safe-access (client and server) helper, `getENV('IS_GOOGLEBOT')`